v19.2/api-stability.md

```diff
@@ -0,0 +1,586 @@
+| `BACKUP` result data files                   | reserved (see note 1 below)               | reserved (see note 1 below)           |
+| `EXPORT` result data files                   | public and programmable                   | reserved                              |
+| `RESTORE`/`IMPORT` SQL parameters and output | public and programmable                   | reserved                              |
+| `RESTORE` input data files                   | reserved (see note 1 below)               | reserved (see note 1 below)           |
+| `IMPORT` input data files                    | public and programmable                   | reserved                              |
+
+Note 1: the output format of `BACKUP` may change between revisions, but
+the following guarantee is preserved: the database and table contents
+that result from `RESTORE`ing a backup into another CockroachDB instance
+will be stable in the same way as other public and programmable
+interfaces.
+
+#### Other SQL statements or constructs
+
+| Component                                                                      | Status if feature documented on this site       | Status if not documented on this site |
+|--------------------------------------------------------------------------------|-------------------------------------------------|---------------------------------------|
+| `EXPLAIN` inputs and outputs                                                   | public and non-programmable                     | reserved                              |
+| `SHOW` inputs                                                                  | public and programmable                         | reserved                              |
+| `SHOW` outputs                                                                 | public and non-programmable                     | reserved                              |
+| `CANCEL`, `PAUSE`, `RESUME` inputs                                             | public and programmable                         | reserved                              |
+| `CANCEL`, `PAUSE`, `RESUME` success/error status                               | public and programmable                         | reserved                              |
+| `CANCEL`, `PAUSE`, `RESUME` outputs                                            | public and non-programmable                     | reserved                              |
+| `SCRUB` inputs and outputs                                                     | public and programmable in “experimental” phase | reserved                              |
+| Tabular data produced by set-generating functions also supported in PostgreSQL | public and programmable                         | public and programmable               |
+| Tabular data produced by set-generating functions specific to CockroachDB      | public and programmable                         | reserved                              |
+
+### SQL introspection interfaces
```

Comment:
@knz @awoods187 
Is this "public and programmable" vs. "reserved" distinction official for all of the `crdb_internal` tables listed here? In addition to listing out the tables in `crdb_internal` with name and description (see #5454), I'd like to have some kind of note about the stability of each table, very similar to the "public and programmable" vs. "reserved" distinction.